### ISSUE TITLE
[Asset Browser] Allow Filtering of _c Files

### ISSUE THREAD

**Issue by JammieDodgerr** [sboxgame/issues#3205]
### For?

S&Box

### What can't you do?

I'd like to be able to filter out compiled files in the asset browser so that I know which files can be opened up and learned from, and which ones cannot.

### How would you like it to work?

A new tickbox under Show Bullshit Files for Show Compiled Files, or a new option in the filter menu in the 3rd column to show Uncompiled files only.

Also there could be padlock icon in the corner of compiled file thumbnails, similar to the downloaded file icon on cached cloud assets.

### What have you tried?

Searching for `t:vmdl -_c` and using regular expressions like `[^_c]`.

**Comment by DoctorGurke** [comment]
somewhat related:  #2364
old asset browser had a little padlock icon on the thumbnail for assets without source. would be cool if we could filter it too

**Comment by JammieDodgerr** [comment]
That's true, the kebab menu on the right probably isnt the best place for it.
I think a filter would be nice though, since there are so few uncompiled source files in core and citizen it would still be hard to find them even with an icon.

**Comment by xezno** [comment]
Sure, don't see why we can't have a filter for it

**Comment by xezno** [comment]
Should be fixed in next build, you can use `@readonly` and `@editable` to filter by assets respectively
